cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,10 @@ Cask Data Application Platform Release Notes
 General
 -------
 
+- The :ref:`File DropZone <cdap-file-drop-zone>` and :ref:`File Tailer <cdap-file-tailer>`
+  are no longer supported as of Release 3.0.
+
```

Comment:
I think this is the only place where we still need to mention drop zone and file tailer.